lambda/src/main/resources/db/migration/V159__add_schemalibraryversion_to_consignment_table.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE "Consignment"
+    ADD COLUMN "SchemaLibraryVersion" text;
```

Comment:
Could you name this column: `MetadataSchemaLibraryVersion` Just to make clear what sort of schema it is.

Reply:
changed